### PR TITLE
User View + Basket Creation View

### PR DESCRIPTION
**Pull request by AleksandraCyp**

### CODE REVIEW

File: src/SharedComponents/BookListItem.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import listProperties from "../SharedFunctions/listProperties";
+
+const BookListItem = ( {book}) => {
+    const { name, author, publisher, year, _id } = book;
```
Comment (lukaszdutka) [inline src/SharedComponents/BookListItem.js]: Maybe deconstruct `{book}` as in Book.js component? :)

File: src/SharedFunctions/addBooksToList.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const addBooksToList = (e, books, chosenBooks) => {
+  if (!e.target.checked) {
+    const newBooks = chosenBooks.filter((book) => book._id !== e.target.id);
+    return newBooks;
```
Comment (lukaszdutka) [inline src/SharedFunctions/addBooksToList.js]:
```suggestion
    return chosenBooks.filter((book) => book._id !== e.target.id);
```

File: src/Views/Basket/Basket.js [hunk 3]
```diff
@@ -1,11 +1,118 @@
...
+  const [basketCreatorVisible, setBasketCreatorVisible] = useState(false);
+  const [chosenPage, setChosenPage] = useState(1);
+  const [offeredPage, setOfferedPage] = useState(1);
+  const onPageLimit = 10;
+
+  useEffect(() => {
+    const getBooks = async () => {
+      const res = await searchMyBooks({
+        Authorization: `Bearer ${accessToken}`,
+      });
```
Comment (lukaszdutka) [inline src/Views/Basket/Basket.js]: why pass whole object with Authorization header, when you can pass only accessToken and the logic associated with header have only inside of `searchMyBooks` method?

**Review by lukaszdutka — APPROVED** [review]
Looks good to me, except some small addressed issues :) 
correct those small things or if you don't want to - feel free to merge it as it is

**Review by memeraki — APPROVED** [review]

**Pull request merged**